### ISSUE TITLE
Bodyslide not working.

### ISSUE THREAD

**Issue by cranedrop** [TanninOne/modorganizer#883]
Fallout 4 - Installed bodyslide and CBBE body and try to run bodyslide. Bodyslide open fine, the problem is there's no outfit/body come up from the drop down menu.

**Comment by krisser143** [comment]
Then you installed the BodySlide addon(s) wrong. Make sure that the structures of the mods are correct. Mine is working fine.

**Comment by RiffyDivine** [comment]
This is a problem with the user and not MO2. Did you forget to activate any mods you wanted to use in bodyslide?

**Comment by cranedrop** [comment]
I've only installed the bodyslide and cbbe body. Nothing showed up

**Comment by alexjet85** [comment]
you need to install "CBBE vanilla outfit" (on CBBE page) or you won't have anything appearing.

**Comment by cranedrop** [comment]
Oh Yay! It works now. Thanks for the help.

**Comment by cranedrop** [comment]
But wondering why must the cbbe vanilla outfit needed?

**Comment by krisser143** [comment]
Because it containts the bodyslide addons. AKA it contains the files to make the clothings bodyslide compatible

**Comment by alexjet85** [comment]
and if you want higher poly outfit try the atomic beauty bodyslide

**Comment by RiffyDivine** [comment]
AB body is nicer looking but very limited in outfit choices.

**Comment by michaelrw10** [comment]
thats always the tradeoff for non-mainstream body meshes.. not worth it imo.. gotta have the matching clothing/armor

**Comment by krisser143** [comment]
You can just download bodyslide compatible clothing and build them for the Atomic Beauty body.

**Comment by alexjet85** [comment]
no, that wouldn't work as atomic beauty is a higher poly model and has it's own sliders in bodyslide

**Comment by krisser143** [comment]
Ah I see, sorry ^^
